src/Munchers/NpmList.spec.ts

```diff
@@ -0,0 +1,156 @@
+
+describe('NpmList SanityCheck', async () => {
+  it('compare prior result to refactored result', async () => {
+    const npmList = new NpmList();
+
+    const coordinates: Array<Coordinates> = await npmList.getInstalledDeps();
+    const coordinatesJson = JSON.stringify(coordinates);
+    expect(coordinatesJson).to.eq(expectedCoordinatesJson);
+  });
+});
+
+describe('NpmList.maybePushNewCoordinate', async () => {
```

Comment:
I don't have a gigantic problem with this, but this is a private method. Generally I'd want to test the publics, and not tie directly to privates (brittle situation). Maybe we need to retinker this class a bit to make it easier to test?

Reply:
Yeah, I felt a little dirty calling "private" directly, but less dirty than trying to test the big ole outer method. Not sure I see a clear path to retinker. I was just happy to manage to cover all the cases. ;)